tests/api.c

```diff
@@ -4256,6 +4256,44 @@ static int test_wolfSSL_CertManagerNameConstraint5(void)
     return EXPECT_RESULT();
 }
 
+static int test_wolfSSL_CRL_duplicate_extensions(void)
+{
+    EXPECT_DECLS;
+#if defined(WOLFSSL_ASN_TEMPLATE) && !defined(NO_CERTS) && \
+    defined(HAVE_CRL) && !defined(NO_RSA) && !defined(WOLFSSL_NO_ASN_STRICT)
+    const unsigned char crl_duplicate_akd[] =
+        "-----BEGIN X509 CRL-----\n"
```

Comment:
The steps for creation of the dulpicate akd CRL need to be documented and I suggest creating a new one not just from the reporter. Document this in the code or in the certs/renewcerts.sh and place into a file. Its fine if this tests requires a filesystem to load the CRL.

Reply:
I actually don't know how to do it. I suspect the reporter had a special tool to create it.